nni/retiarii/converter/op_types.py

```diff
@@ -1,18 +1,31 @@
 MODULE_EXCEPT_LIST = ['Sequential']
-RETIARII_BASE_OPS = ['Placeholder']
+
 
 class Type:
```

Comment:
Why use a class here?

Reply:
will change class name and split this class into two classes

Reply:
let me refactor this part in the next pr